webpack.config.js

```diff
@@ -31,12 +35,24 @@ let config = {
     ],
   },
   plugins: [
-    new webpack.BannerPlugin({
-      banner: `Pushin.js - v${PACKAGE.version}\nAuthor: ${PACKAGE.author}\nLicense: ${PACKAGE.license}`,
-    }),
```

Comment:
@arturovt - Nitpick: I personally like having the banner because it helps me verify that the documentation site is using the correct version. Any way we could add this back in to the output as well?

Reply:
Should the banner exist in the library itself or in the docs app too?

Reply:
I think the docs app is most important, since anyone using the library knows the version number they've installed by checking their package.json. What do you think?

Reply:
Alright, I’ll get it back.

Reply:
Done